pom.xml

```diff
@@ -207,23 +232,22 @@
 
         <!-- Java decompiler -->
         <dependency>
-            <groupId>org.bitbucket.mstrobel</groupId>
-            <artifactId>procyon-core</artifactId>
-            <version>${procyon.version}</version>
+            <groupId>org.jd</groupId>
+            <artifactId>jd-core</artifactId>
+            <version>${jd-core.version}</version>
         </dependency>
-
         <dependency>
-            <groupId>org.bitbucket.mstrobel</groupId>
-            <artifactId>procyon-reflection</artifactId>
-            <version>${procyon.version}</version>
+            <groupId>org.benf</groupId>
+            <artifactId>cfr</artifactId>
+            <version>${cfr.version}</version>
         </dependency>
-
         <dependency>
-            <groupId>org.bitbucket.mstrobel</groupId>
-            <artifactId>procyon-compilertools</artifactId>
-            <version>${procyon.version}</version>
+            <groupId>org.jetbrains.java.decompiler</groupId>
+            <artifactId>fernflower</artifactId>
+            <version>1.0</version>
+            <scope>system</scope>
+            <systemPath>${project.basedir}/jars/fernflower-decompiler.jar</systemPath>
```

Comment:
Should it be `jar-dependencies` ?

Reply:
Nope pick any name you'd like for me naming is the most difficult part :)

Reply:
Sorry just fixed it didn't notice it was not renamed in the pom. Thought you were providing a different name suggestion.